src/kotlin_script.h

```diff
@@ -12,9 +12,16 @@ class KotlinScript : public Script {
     friend class KotlinInstance;
     friend class TypeManager;
 
+    enum AccessMode {
+        NONE,
+        PATH,
+        NAME,
+    };
```

Comment:
I personally prefer to keep using Path/Name in the code instead of directly writing KotlinScript and RegistrationScript or similar. It just feels more natural this way. 
The hierarchy of logic is the following:
```
Script
   JVMSCript
      PathScript
          KotlinScript
          JavaScript
      NamedScript
          GDJScript
```
Logic related to paths and names are above languages or file types. It just happens that, for now, there is only one type of Pathscript and one type of NamedScript. It will change when we add Java support.

Reply:
Ok for me